distribution/docs/src/main/resources/_contents/_configuring/configuring-global-settings-contents.adoc

```diff
@@ -103,30 +103,48 @@ Does not change the address the system runs on.
 |HTTPS Port
 |org.codice.ddf.system.httpsPort
 |String
-|The https port used by the system.
+|The https port used to advertise the system.
 
-NOTE: This *DOES* change the port the system runs on.
+NOTE: This does not change the port the system runs on.
 |8993
 |Yes
 
 |HTTP Port
 |org.codice.ddf.system.httpPort
 |String
-|The http port used by the system.
+|The http port used to advertise the system.
 
-NOTE: This *DOES* change the port the system runs on.
+NOTE: This does not change the port the system runs on.
 |8181
 |Yes
 
 |Default Port
 |org.codice.ddf.system.port
 |String
-|The default port used to advertise the system. This should match either the http or https port.
+|The default port used to advertise the system. This should match either the above http or https port.
 
 NOTE: Does not change the port the system runs on.
 |8993
 |Yes
 
+|Internal HTTPS Port
```

Comment:
To Eric's comment, I think that would be unnecessary if we change the external port properties to be set to the internal port property values by default. I think that pretty clearly spells it out for admins.

To Mike's comment, I agree that we should probably document that somewhere and I'm wondering if that would be taken care of by our HA guide.